InventoryCatalog/Model/ResourceModel/BulkInventoryTransfer.php

```diff
@@ -127,7 +127,7 @@ private function transferInventory(
 
         if ($orgSourceItem !== null) {
             $status = (int) $orgSourceItem[SourceItemInterface::STATUS];
-        } elseif ($dstSourceItemQty !== null) {
```

Comment:
Hello @dennisvanderweide,

Thank you for your contribution!

Please let us know in which scenario the existing will create an issue.

Also please help us with the manual testing scenario.

Thank you